source/operators/operator-library.yaml

```diff
@@ -1020,6 +1073,62 @@ Resources:
           DATAPLANE_BUCKET: !Ref "DataPlaneBucket"
           botoConfig: '{"user_agent": "aws-tm-mie/python3.7/lambda"}'
 
+  startCustomLabelDetection:
+    Type: "AWS::Lambda::Function"
+    Properties:
+      Handler: "start_custom_label_detection.lambda_handler"
+      Layers:
+        - !Ref MediaInsightsEnginePython37Layer
+      Role: !GetAtt "rekognitionRole.Arn"
+      Code:
+        S3Bucket: !FindInMap ["SourceCode", "General", "S3Bucket"]
+        S3Key:
+          !Join [
+            "/",
+            [
+            !FindInMap ["SourceCode", "General", "KeyPrefix"],
+            "start_custom_label_detection.zip",
+            ],
+          ]
+      Runtime: "python3.7"
+      Environment:
+        Variables:
+          REKOGNITION_SNS_TOPIC_ARN: !Ref "snsRekognitionTopic"
+          REKOGNITION_ROLE_ARN: !GetAtt "rekognitionRole.Arn"
+          PROJECT_VERSION_ARN: "arn:aws:rekognition:us-east-1:051545784337:project/AnimatedCharacters/version/AnimatedCharacters.2020-02-29T10.49.44/1582994985715"
```

Comment:
Instead of using an environment variable, I think the model_arn should be provided by the user via the web form for configuring the workflow in UploadToAWSS3.vue.

Reply:
I updated the startCustomLabelDetection to check for the right image media type. How do I add it to this PR ? 
Also, can you suggest how to update the .vue to input the model arn from the user ?
@ianwow